src/components/sbb-option/sbb-option.scss

```diff
@@ -31,14 +43,27 @@
 
 :host([data-group-disabled]),
 :host([disabled]:not([disabled='false'])) {
-  --sbb-option-color: var(--sbb-color-graphite-default);
+  --sbb-option-color: var(--sbb-color-metal-default);
```

Comment:
It should be granite I think?
```suggestion
  --sbb-option-color: var(--sbb-color-granite-default);
```

Reply:
<img width="746" alt="Bildschirmfoto 2023-05-25 um 15 49 54" src="https://github.com/lyne-design-system/lyne-components/assets/29278232/5d2462da-0b5f-48a2-8357-35ef56c89dfb">